### ISSUE TITLE
Some strings are checked in TypeScript when they shouldn't

### ISSUE THREAD

**Issue by borekb** [bartosz-antosik/vscode-spellright#80]
My config:

```
    "spellright.language": "en",
    "spellright.addToSystemDictionary": true,
    "spellright.notificationClass": "information",
    "spellright.documentTypes": [
        "markdown", "typescript"
    ],
    "spellright.spellContext": "body comments strings",
    "spellright.spellContextByClass": {
        "markdown": "body",
        "typescript": "comments",
        "json": "comments",
        "html": "body"
    },
```

Some TS strings are checked, but only the first two words, _not_ the misspelled string at the bottom:

![image](https://user-images.githubusercontent.com/101152/31740464-bc8ca5a4-b451-11e7-8bf5-0fafc4bc2886.png)

**Comment by bartosz-antosik** [comment]
Could you please paste the part of the document - would be easier to test it.

**Comment by borekb** [comment]
I'm able to reproduce on this snippet:

```
import * as execa from 'execa';
import * as path from 'upath';
import * as chalk from 'chalk';
import * as yargs from 'yargs';
import * as common from './common';
import * as infoCommand from './info';
import { logger } from '../logger';

const demo = 'asdasdasdaa';
```

**Comment by bartosz-antosik** [comment]
Thank you.

**Comment by bartosz-antosik** [comment]
Could you please screenshot for me diagnostic messages for this file?

**Comment by borekb** [comment]
Do you mean this?

![image](https://user-images.githubusercontent.com/101152/31744250-63348c12-b45e-11e7-9be4-5cc5035800a7.png)

**Comment by bartosz-antosik** [comment]
Yes, this, thank you. Incredible. Works totally different in my case.

**Comment by borekb** [comment]
I can reproduce on other files as well:

![image](https://user-images.githubusercontent.com/101152/31854912-bdbeee04-b6a0-11e7-9b96-8c2231d3aa44.png)

**Comment by bartosz-antosik** [comment]
Yes, thank you. I should have wrote something: It's been reproduced & fixed. Just it has to wait a moment for a next release or maybe I will do some order & release just this fix. Please bit more patience.

**Comment by borekb** [comment]
Sure, no problem, I just thought that it's still not clear where the problem is so wanted to confirm it. No problem waiting 👍 .

**Comment by bartosz-antosik** [comment]
Should be fixed in 1.2.24. Please let me know - this was a fix touching quite a deal of things.

**Comment by borekb** [comment]
Works nicely, thanks!
